[CI] GenerativeForkIT class failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #9269 / ubuntu-2404_platform-support-unix](https://gradle-enterprise.elastic.co/s/bo4wk7g2woxxa)
- [elasticsearch-periodic-platform-support #9269 / almalinux-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/sopbpm2thendi)
- [elasticsearch-periodic-platform-support #9269 / rhel-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/ysgkomd3kgod6)
- [elasticsearch-periodic-platform-support #9269 / windows-2025_checkpart3_platform-support-windows](https://gradle-enterprise.elastic.co/s/mzqpuxkvuiqei)
- [elasticsearch-periodic-platform-support #9269 / almalinux-8-aarch64_checkpart3_platform-support-arm](https://gradle-enterprise.elastic.co/s/2e36q77jxdglk)
- [elasticsearch-pull-request #79834 / part-3](https://gradle-enterprise.elastic.co/s/sjw5bgt34ci66)
- [elasticsearch-pull-request #79818 / part-3](https://gradle-enterprise.elastic.co/s/y6rqth32yr7le)
- [elasticsearch-pull-request #79814 / part-3](https://gradle-enterprise.elastic.co/s/uorfqmn5ufei2)
- [elasticsearch-pull-request #79801 / part-3-fips](https://gradle-enterprise.elastic.co/s/q4zog7lah32hw)
- [elasticsearch-pull-request #79800 / part-3](https://gradle-enterprise.elastic.co/s/dym3ud5hdctic)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT" -Dtests.method="test {folding.LocalSourceMultiAttributeString SYNC}" -Dtests.seed=C97C96246A62AB6E -Dtests.locale=ko-KP -Dtests.timezone=Europe/Belgrade -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: circuit breakers not reset to 0Expected a map containing
estimated_size_in_bytes: expected <0> but was <96>
         estimated_size: expected "0b" but was "96b"
               overhead: <1.0> unexpected but ok
    limit_size_in_bytes: <322122547> unexpected but ok
             limit_size: "307.1mb" unexpected but ok
                tripped: <0> unexpected but ok
```

**Issue Reasons:**
- [main] 4 consecutive failures in step windows-2025_checkpart3_platform-support-windows
- [main] 4 consecutive failures in step almalinux-8-aarch64_checkpart3_platform-support-arm
- [main] 39 failures in class org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT (6.6% fail rate in 593 executions)
- [main] 15 failures in step part-3 (5.0% fail rate in 301 executions)
- [main] 2 failures in step almalinux-9_platform-support-unix (28.6% fail rate in 7 executions)
- [main] 4 failures in step windows-2025_checkpart3_platform-support-windows (50.0% fail rate in 8 executions)
- [main] 4 failures in step almalinux-8-aarch64_checkpart3_platform-support-arm (50.0% fail rate in 8 executions)
- [main] 2 failures in step windows-2022_checkpart3_platform-support-windows (28.6% fail rate in 7 executions)
- [main] 2 failures in step ubuntu-2404-aarch64_checkpart3_platform-support-arm (28.6% fail rate in 7 executions)
- [main] 6 failures in pipeline elasticsearch-periodic-platform-support (75.0% fail rate in 8 executions)
- [main] 15 failures in pipeline elasticsearch-pull-request (5.2% fail rate in 288 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -299,12 +299,6 @@ tests:
 - class: org.elasticsearch.compute.operator.LimitOperatorTests
   method: testEarlyTermination
   issue: https://github.com/elastic/elasticsearch/issues/128721
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT
-  method: test {csv-spec:lookup-join.EnrichLookupStatsBug}
-  issue: https://github.com/elastic/elasticsearch/issues/129228
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT
-  method: test {lookup-join.MultipleBatches*
-  issue: https://github.com/elastic/elasticsearch/issues/129210
 - class: org.elasticsearch.entitlement.runtime.policy.FileAccessTreeTests
   method: testWindowsMixedCaseAccess
   issue: https://github.com/elastic/elasticsearch/issues/129167

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -465,9 +465,6 @@ tests:
 - class: org.elasticsearch.xpack.ml.integration.RevertModelSnapshotIT
   method: testRevertModelSnapshot
   issue: https://github.com/elastic/elasticsearch/issues/132733
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT
-  method: test {csv-spec:lookup-join.MvJoinKeyFromRowExpanded}
-  issue: https://github.com/elastic/elasticsearch/issues/132778
 - class: org.elasticsearch.gradle.internal.transport.TransportVersionManagementPluginFuncTest
   method: definitions have primary ids which cannot change
   issue: https://github.com/elastic/elasticsearch/issues/132788

